int32 pallet_number
string operation
---
bool success

